psi4/src/psi4/libfock/CMakeLists.txt

```diff
@@ -32,3 +32,10 @@ target_link_libraries(fock
   PRIVATE
     gau2grid::gg
   )
+
+if(ENABLE_BrianQC)
+  target_compile_definitions(fock
+    PUBLIC
+      USING_BrianQC
+    )
+endif()
```

Comment:
When the CMake target is available, should be able to do

```
if(TARGET BrianQC::BrianQC)
  target_link_libraries(fock
    PUBLIC
      BrianQC::BrianQC
    )
endif()
```

Reply:
You are right, I will rewrite the conditional linking to match the CMake style/pattern used in Psi4.

Reply:
I have rewritten the conditionals to follow Psi4's conventions, please let me know if anything else is amiss.

Reply:
As far as I know, the build speed is not affected. However, most of the brian-specific code blocks use one or two of BrianQC's own datatypes (for example, BrianCookie and brianInt), declared in our headers. Thus, when building Psi4 without BrianQC, almost all of our code has to be disabled, otherwise it will not compile at all (referring to unavailable types). A possible workaround would be declaring a dummy version of those types when building without BrianQC (adding an #else to the #ifdefs that include BrianQC's headers), however, I'm not sure that would be a nicer solution than leaving the #ifdefs in. What is your opinion, which method introduces less ugliness into the code?

Reply:
The above suggestion is just what I was thinking of. It compiles fine with BrianQC disabled for me. It's a fairly minor detail, so at your choice.